com.seafile.Client.yaml

```diff
@@ -0,0 +1,83 @@
+app-id: com.seafile.Client
+runtime: org.kde.Platform
+runtime-version: '5.15'
```

Comment:
```suggestion
runtime-version: '5.15-21.08'
```

Reply:
Fails to build on `5.15-21.08`, it looks like there is a regression and some path is not set correctly:

```
----------------------------------------------------------------------
 /usr/bin/mkdir -p '/app/include'
 /usr/bin/install -c -m 644 searpc-client.h searpc-server.h searpc-utils.h searpc.h searpc-named-pipe-transport.h '/app/include'
make[2]: Leaving directory '/run/build/libsearpc/lib'
make[1]: Leaving directory '/run/build/libsearpc/lib'
Making install in pysearpc
make[1]: Entering directory '/run/build/libsearpc/pysearpc'
make[2]: Entering directory '/run/build/libsearpc/pysearpc'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/usr/lib/python3.9/site-packages/pysearpc'
/usr/bin/mkdir: cannot create directory ‘/usr/lib/python3.9/site-packages/pysearpc’: Read-only file system
make[2]: *** [Makefile:338: install-pysearpcPYTHON] Error 1
make[2]: Leaving directory '/run/build/libsearpc/pysearpc'
make[1]: *** [Makefile:431: install-am] Error 2
make[1]: Leaving directory '/run/build/libsearpc/pysearpc'
make: *** [Makefile:476: install-recursive] Error 1
Error: module libsearpc: El proceso hijo terminó con el código 2
```

Reply:
Fixed by manually adding `--with-python_prefix=/app`